requirements/base.in

```diff
@@ -0,0 +1,14 @@
+# Main requirements of your plugin application.
+-c constraints.txt
+-r github.in
+
+celery
+Django
+djangorestframework
+django-filter
+django-oauth2-provider
+django-waffle
+edx-proctoring
+edx-opaque-keys
```

Comment:
setup.py has some of these dependencies, should we remove them from there? or change setup.py to use this as dependencies?

In case of the second, I think that we need to change this with edx-opaque-keys[django] in order to prevent error when this package is installed using `pip install eox-core`